usdevs/usc-website
# Compressed images for 20th USC MC

## Description of Change
Compressed photos for the new MC and removed unused folders containing uncompressed media so that the hosting fees don't fly up too much (":

## Link To Issue 
NA

## How to test this change
Run a local server to check the site is up and running! The About us page should be populated with the latest info.

## Checklists
### Checklist for developer of feature
- [ ] I have tested my code against the Travis CI, and it has passed.

### Checklist for reviewers
- [ ] I have followed the given steps to test the PR, and it passes.
